package.json

```diff
@@ -40,6 +40,9 @@
     "singleQuote": true,
     "trailingComma": "all"
   },
+  "dependencies": {
+    "js-cookie": "^2.2.1"
+  },
```

Comment:
Added this rather than hand-cut our own JS-wrangling cookies. Cross-browser tested and seems fine.

Reply:
(MIT licence)